tests/run/fused_cpdef.pyx

```diff
@@ -3,14 +3,12 @@ import sys, io
 
 cy = __import__("cython")
 
-cpdef func1(self, cython.integral x, printto=None):
-    if printto is None:
-        printto = sys.stdout
-    print >> printto, "%s," % (self,),
+cpdef func1(self, cython.integral x):
+    print "%s," % (self,),
```

Comment:
This isn't what I meant, it's still Py2-only code. I suggest using an f-string and `print(f"…")`.